actions/deploy/entrypoint.sh

```diff
@@ -21,17 +23,33 @@ fi
 
 # Inject "image" as a template variable to a new copy of the vars file.
 # If the file doesn't exist, it is created. The original file is left untouched.
-if [ ! -z "$IMAGE" ]; then
+if [ -n "$IMAGE" ]; then
     export VARS_ORIGINAL="$VARS"
-    export VARS=`mktemp`
+    VARS=$(mktemp)
+    export VARS
     if [ -z "$VARS_ORIGINAL" ]; then
-        echo "---" > $VARS
+        echo "---" > "$VARS"
     else
-        cat $VARS_ORIGINAL > $VARS
+        cat "$VARS_ORIGINAL" > "$VARS"
     fi
-    yq w --inplace $VARS image "$IMAGE"
+    yq w --inplace "$VARS" image "$IMAGE"
+fi
+
+
+# if no apikey is set, use use the id-token to get a jwt token for the deploy CLI
```

Comment:
Any reason why this is not implemented in the CLI itself? Usually prefer go code over bash.